src/iosrv/init.c

```diff
@@ -587,6 +671,7 @@ Options:\n\
       Print this description\n",
 		prog, prog, modules, daos_sysname, dss_storage_path,
 		dss_socket_dir, dss_nvme_conf);
+	exit(0);
```

Comment:
Was this added intentionally?

Reply:
I did.  It looked like an oversight that it would not exit after display the help text.  If people disagree, I can remove it.

Reply:
Removed.